app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/screen/TeleconsultRecordInit.kt

```diff
@@ -0,0 +1,12 @@
+package org.simple.clinic.teleconsultlog.teleconsultrecord.screen
+
+import com.spotify.mobius.First
+import com.spotify.mobius.Init
+import org.simple.clinic.mobius.first
+
+class TeleconsultRecordInit : Init<TeleconsultRecordModel, TeleconsultRecordEffect> {
+
+  override fun init(model: TeleconsultRecordModel): First<TeleconsultRecordModel, TeleconsultRecordEffect> {
+    return first(model, LoadTeleconsultRecordWithPrescribedDrugs(model.teleconsultRecordId))
```

Comment:
Don't we need to check for the screen restoration use case?

Reply:
Yes, but I couldn't figure out which things to check in this particular scenario. Because there will be default values in the model from start. So we cannot just verify if it's null. Any ideas on how we can handle these scenarios?

Reply:
Ah, I see what you mean. Let's leave it as is for now and we can revisit this later.